rest-service/manager_rest/storage/models_base.py

```diff
@@ -216,6 +216,15 @@ def _get_identifier_dict(self):
     def unique_id(cls):
         return 'id'
 
+    @classmethod
+    def default_sort_column(cls):
+        """If no sort is requested, order by the column specified by this.
+
+        This is so that requests with pagination make sense even with no
+        sort requested by the user.
+        """
+        return getattr(cls, cls.unique_id())
```

Comment:
Does `cls.unique_id()` translate to a DB-proprietary sequence?

Reply:
it returns whatever the class wants to return, in this case `id` (SQLModelBase) or `_storage_id` (SQLResourceBase) (so for now, yes)